integration-tests/actions/keeper_benchmark_helpers.go

```
@@ -93,35 +93,40 @@ func ResetUpkeeps(
 	checkGasToBurn, // How much gas should be burned on checkUpkeep() calls
 	performGasToBurn, // How much gas should be burned on performUpkeep() calls
 	firstEligibleBuffer int64, // How many blocks to add to randomised first eligible block
-	predeployedContracts []string,
+	predeployedContracts []contracts.KeeperConsumerBenchmark,
 	upkeepResetterAddr string,
 ) {
-	contractLoader, err := contracts.NewContractLoader(client)
+	contractLoader, _ := contracts.NewContractLoader(client)
```

Why ignore this error?